code/modules/mod/modules/modules_general.dm

```diff
@@ -655,3 +655,108 @@
 		has_tk = FALSE
 		return
 	mod.wearer.dna.remove_mutation(TK)
+
+//Special module for the Captain's Magnate suit, and the CC Corporate suit.
+//Literally only a thing so that https://pipe.miroware.io/5b52ba1d94357d5d623f74aa/mspfa/Nuke%20Ops/Panels/0648.gif can be real
+/obj/item/mod/module/hat_stabilizer
+	name = "MOD hat stabilizer module"
+	desc = "A simple set of deployable stands, directly atop one's head; \
+		these will deploy under a select few hats to keep them from falling off, allowing them to be worn atop the sealed helmet. Unfortunately, you still need to take off the hat first to deploy the helmet itself. \
+		This is a must-have for Nanotrasen Captains, enabling them to show off their authoritative hat even while in their MODsuit."
+	icon_state = "hat_holder"
+	module_type = MODULE_PASSIVE //The description says it auto-deploys for certain hats; that's just an excuse to have a whitelist of items.
+	complexity = 0 //Its FREE!
+	active_power_cost = 0 //Yes, thats right, FREE!
```

Comment:
active power cost doesnt even work this is a passive module
and complexity is inherited from parent

Reply:
Figured it'd be inherited, realized after reading this that my references were all subtypes of already-existing modules. I'll fix this